src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -1339,9 +1339,8 @@ internal static bool IsAttributeDefined(RuntimeModule decoratedModule, int decor
             RuntimeModule decoratedModule, int decoratedMetadataToken, int pcaCount, RuntimeType? attributeFilterType)
         {
             RuntimeType.ListBuilder<object> attributes = new RuntimeType.ListBuilder<object>();
-            RuntimeType.ListBuilder<object> _ = default;
 
-            AddCustomAttributes(ref attributes, decoratedModule, decoratedMetadataToken, attributeFilterType, false, ref _);
+            AddCustomAttributes(ref attributes, decoratedModule, decoratedMetadataToken, attributeFilterType, false, new RuntimeType.ListBuilder<object>());
```

Comment:
I assume this change to use explicit ctor wasn't necessary for the issue - they both pass in a temp struct.

Reply:
Correct. I could have kept the local around. (The local was required to pass by ref, but it is not required to pass by value.)